app/Http/Controllers/Auth/LoginController.php

```diff
@@ -179,8 +177,8 @@ public function login(Request $request)
         }
 
         if ($user = Auth::user()) {
-            $user->last_login = \Carbon::now();
-            \Log::debug('Last login:'.$user->last_login);
+            $user->last_login = Carbon::now();
+            Log::debug('Last login:'.$user->last_login);
```

Comment:
We can probably actually pull this log line

Reply:
My IDE screams at me because it cannot resolve the \Log and \Carbon - so yes, its not _needed_ but should be changed some day.